Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the custom base image with Ruby, Node.js, npm, and Shopify CLI pre-installed
+FROM aaronmedinaaligent/shopify-cli-base:latest
+
+# Set build arguments for user ID and group ID
+ARG USER_ID
+ARG GROUP_ID
+
+# Create a group and user matching the host UID and GID to avoid permission issues
+RUN groupadd -g $GROUP_ID shopifygroup && \
```

Comment:
This will need `-l` or `--no-log-init` because of a fun big in linux. Currently building this locally _might_ crash and fill out your local disk.

https://github.com/moby/moby/issues/5419

Reply:
I will add this, thanks for the reference.
Good call